eip155/README.md

```diff
@@ -0,0 +1,40 @@
+---
+namespace-identifier: eip155
+title: EIP155 Namespace, aka EVM Chains
+---
+
+# Namespace for EIP155 chains
+
+This document describes the syntax and structure of the [EIP155][] namespace,
+commonly referred to as the "EVM" namespace for chains using the Ethereum
+Virtual Machine. 
+
+## Introduction
+
+[EIP155][] is an Ethereum Improvement Process specifying addressing across
+Ethereum-based chains, generalized by [CAIP-2][]. Definition of valid
+`references` within that namespace and constraints are deferred to the
+[EIP155][] specification, which may be superseded by future EIPs.
+
+## References
```

Comment:
I think the markdown of this section is not correctly formatted.

Reply:
see below! the first part is a bulleted list, the second part is invisible in the rendered markdown